includes/discovery/processors/cambium.inc.php

```diff
@@ -21,7 +21,7 @@
             '0',
             'cambium-cpu',
             $descr,
-            '100',
+            '10',
```

Comment:
This value is the "processor_precision" value in the database. includes/polling/processors.inc.php divides whatever the CPU-specific polling module returns by this number.

The raw value presented by the OID ranges from 0-1000 - a value of 50 is 5% - so the raw value needs to be divided by 10.

Currently, includes/polling/processors/cambium-cpu.inc.php divides the raw value by 10, and passes that value back to includes/polling/processors.inc.php;

https://github.com/librenms/librenms/blob/149857f246fef4d83f5f37a7e5a87fdedce113f2/includes/polling/processors/cambium-cpu.inc.php#L14-L16

processors.inc.php then divides the return value by processor_precision;

https://github.com/librenms/librenms/blob/149857f246fef4d83f5f37a7e5a87fdedce113f2/includes/polling/processors.inc.php#L27

Giving us a total divisor of 1000, which we see in a poller run;

```
Processor Processor... Cambium CPU UsageSNMP[/usr/bin/snmpget -v2c -c COMMUNITY -OvQ -M /opt/librenms/mibs:/opt/librenms/mibs/cambium:/opt/librenms/mibs/cambium udp:HOSTNAME:161 CAMBIUM-PMP80211-MIB::sysCPUUsage.0]
50
 
0.05%
```

(0.05 being 1/1000th of 50)

This gives us two options;

1. Remove the division from cambium-cpu.inc.php, and set processor_precision to 10
2. Leave the division in cambium-cpu.inc.php, and set processor_precision to 1

I've opted for option 1 - a search indicates processor_precision is only used by processors.inc.php, and only for dividing the raw value returned from a sub-module, so it seems a bit redundant to move it out to the device-specific sub-module when functionality is already in place.

If there's a compelling reason to choose option 2, please let me know - the only reason I can think of is that it's modifying two files instead of one, which doesn't seem like a great reason.